### DISCUSSION TITLE
Add TEXT on photo

### DISCUSSION THREAD

**Discussion by satimis** [ImageMagick/ImageMagick#5262] [General]
Ubuntu 20.04

Adding text on photo
================

I run following command on Terminal to add "Today June 22. 2022" on photo

`$ convert -pointsize 70 -fill yellow -draw 'text 400,150 "Today June 22. 2022"' input_photo.jpg  output_photo.jpg`

It works seamlessly.

Please advise how to add Chinese characters on photo.  Thanks

Regards

**Comment by fmw42** [comment]
Best I can suggest is to see https://imagemagick.org/Usage/text/#unicode. Use unicode in a UTF-8 file.

**Reply by satimis** [reply]
Thanks for your advice and link.
I got it done.  I have to define the FONT;
-font Noto-Sans-CJK-HK

`$ convert -pointsize 70 -fill yellow -font Noto-Sans-CJK-HK -draw 'text 400,150 "拍攝於 1961"' input_photo.jpg  output_photo.jpg
`
How about adding a logo center-above the TEXT?

Regards

**Reply by fmw42** [reply]
Create a text image on transparent background using label:"some text".  Then append that below the logo image. Then take the appended image and composite that over your larger input image at the location and gravity you want.

**Reply by satimis** [reply]
I already have the logo.jpg image.  But I have no idea adding it to the convert command line and resize it simultaneously?

Regards

**Reply by fmw42** [reply]
Post your input image and logo image so I can work with them.
